types/summernote/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "dtslint/dt.json",
```

Comment:
For new packages, this file should only `"extends": "dtslint/dt.json"`

Reply:
Hi,

I have this declaration:
```
summernote(command: 'destroy'): JQuery;
summernote(command: 'code', markupStr?: string): JQuery;
summernote(command: 'editor.pasteHTML' | 'pasteHTML', markup: string): JQuery;
```

Without the following tslint.json config, the declaration causes an error
```
"unified-signature": false
```
However, I intentionally don't want to combine them together for the sake of documentation.
The library works in a funny way that when command is certain string, some parameters are needed/not-needed/optional. (i.e. the declaration above)

How should I proceed with this?

Thanks

Reply:
Can you try to disable the rule in source code like `// tslint:disable-next-line unified-signature`

Reply:
There are 43 lines with non-unified signatures. Should I inline each one of them? Is there any other acceptable workaround?

Reply:
You can try `// tslint:disable unified-signature`